CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Changelog
 
-# [7.3.3] - 2024-07-15
+## [7.3.4]
+
+### Fixed
+- CANopen multiple drive connection
```

Comment:
It only happened when using a Kvaser, right?

Reply:
Yes, you are right, I will add it